packages/xo-web/src/xo-app/vm/tab-console.js

```diff
@@ -222,15 +227,28 @@ export default class TabConsole extends Component {
             </div>
           </Col>
           <Col mediumSize={2} className='hidden-lg-down'>
-            <input
-              className='form-control'
-              max={3}
-              min={0.1}
-              onChange={this.linkState('scale')}
-              step={0.1}
-              type='range'
-              value={scale}
-            />
+            <Row>
+              <Col mediumSize={8}>
+                <input
```

Comment:
Did the original demand from this PR was for the percentage to be editable?

If not, why not simply display the percentage next to the slider as a text?